ProductsWebApi/Services/JwtAuthService.cs

```diff
@@ -52,26 +54,64 @@ public AuthToken GetToken(Models.Json.User user)
             return authToken;
         }
 
-        private ClaimsIdentity GetIdentity(Models.Json.User verifiedUser)
+        public async Task AddUser(User user)
+        {
+            byte[] salt = GetPseudorandomByteArray();
+            string password = GetHashString(user.Password, salt);
+
+            await _userService.Add(new Models.Entities.UserEntity {
+                Name = user.Username,
```

Comment:
заменить название свойства:
```suggestion
                Name = user.Name,
```

Reply:
исправлено